app/dashboard/templates/profiles/header_details.html

```diff
@@ -0,0 +1,71 @@
+                    <i class="far fa-star {% if avg_rating.overall != 0 and i <= avg_rating.overall %}fas{% endif %}"></i>
+                  {% endfor %}
+                  <small class="text-muted">({{ avg_rating.total_rating }} {% trans "rating" %}{{avg_rating.total_rating | pluralize}})</small>
+                </div>
+                <p class="profile-header__links clearfix mt-1">
+                  <a href="{{ profile.data.html_url }}?tab=repositories" target="_blank" rel="noopener noreferrer" >
+                    <i class="fab fa-github"></i>
+                  </a>
+                  {% if profile.data.blog and user.is_authenticated %}
+                    <a href="{{ profile.data.blog|add_url_schema }}" target="_blank" rel="noopener noreferrer" >
+                      <i class="fas fa-home"></i>
+                    </a>
+                  {% endif %}
+                  {% if profile.linkedin_url and user.is_authenticated %}
+                    <a href="https://linkedin.com/in/{{ profile.linkedin_url }}" target="_blank" rel="noopener noreferrer" >
+                      <i class="fas fa-linkedin"></i>
+                    </a>
+                  {% endif %}
+                  {% if profile.data.email and user.is_authenticated %}
+                    <a href="mailto:{{profile.data.email | obfuscate}}">
+                      <i class="far fa-envelope"></i>
+                    </a>
+                  {% endif %}
+                </p>
+
+              {% if preferred_payout_address %}
+
+
+                <div class="my-2">
+                  <svg xmlns="http://www.w3.org/2000/svg" width="10" height="16" class="mr-2 mb-1"><path fill="#000" fill-rule="nonzero" d="M10 8.113L5.002 11 0 8.113 5.002 0 10 8.113zm-5 3.821L0 9l5 7 5-7-5 2.934z"/></svg>
+                  <span id="preferred-address" data-truncatehash title="{{ preferred_payout_address }}">{{ preferred_payout_address }}</span>
+                </div>
+              {% else %}
+                <div class="my-2" id="preferred-address">
+                  {% if user.is_authenticated and request.path == '/profile/' %}
+                    <a href="/settings/account" class=""><i class="fas fa-plus-circle"></i> Add a preferred address</a>
+                  {% else %}
+                    No preferred address
+                  {% endif %}
+                </div>
+              {% endif %}
+              {% if  profile.show_job_status %}
+                <div class="my-2" id="job_status">
+                  <i class="fa fa-briefcase mr-2" aria-hidden="true"></i>{{ profile.job_status_verbose }}
+                  {% if profile.resume %}<a href="{{ MEDIA_URL }}{{profile.resume}}" download>View Resume</a>{% endif %}
+                </div>
+              {% endif %}
+              {% if profile.data.bio %}
+                <div class="profile-header__bio my-2">
+                  {{ profile.data.bio }}
+                </div>
+              {% endif %}
+            </div>
```

Comment:
^ @owocki would you be open to setting up a linter into the editor ? 😅

Reply:
i have anaconda setup. i think thats python only.  i guess `make fix` doesnt lint html?